### ISSUE TITLE
Distinguish types of blog posts on `blog`

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/scholarslab.github.io#1045]
The blog listing has multiple types (essay, announcement, update), each currently with a different layout and structure.

**Comment by amandavisconti** [comment]
I've got #1015 for a /blog visual way to distinguish what type of post something is. Is this for the same thing?

If not and it's for their layout/structure, some info:

`category: essay` and `category: announcement` use the same post.html template and /blog page display (announcements don't use tags)

'category: update` uses the update.html template (not linked from homepage "recent activity" list, as there is no important additional info there). 'category: update` posts are included in the /blog page lists of posts, but are styled slightly different: 
Title is: [DATE] update: then hyperlinked [~3 word descriptor, like "staff conference activity"]
Instead of excerpt, full 340-character or fewer text displayed (only if in top 4 most recent posts)
individual 'category: update` posts display the text, plus if I used YAML to list involved staff adds that (also will appear on the people named's bio page list of "blog posts by or lab updates about"
